app/hooks/useActiveClaimCondition.tsx

```
@@ -0,0 +1,45 @@
+import {
+  getActiveClaimCondition,
+  GetActiveClaimConditionParams,
+} from 'thirdweb/extensions/erc1155';
+
+type ActiveClaimCondition = {
+  contract: Readonly<ContractOptions<any>>;
+} & GetActiveClaimConditionParams;
+
+export function useActiveClaimCondition(props: ActiveClaimCondition) {
+  const [activeClaimCondition, setActiveClaimCondition] = useState<
+    ActiveClaimCondition | {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)